src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj

```diff
@@ -16,6 +16,7 @@
   </PropertyGroup>
   <ItemGroup Label="Project References">
     <ProjectReference Include="..\..\..\Compilers\Core\Portable\Microsoft.CodeAnalysis.csproj" />
+    <ProjectReference Include="..\..\..\Compilers\CSharp\Portable\Microsoft.CodeAnalysis.CSharp.csproj" />
```

Comment:
Remove this. `Microsoft.CodeAnalysis.Features` should not have any dependencies on C#. C# specific functionality should go in `Microsoft.CodeAnalysis.Features.CSharp`. I assume a add usings light builb added this project reference for you :)

Reply:
Removed